render/hyperscript.js

```diff
@@ -31,7 +31,7 @@ function compileSelector(selector) {
 
 function execSelector(state, attrs, children) {
 	var hasAttrs = false, childList, text
-	var className = attrs.className || attrs.class
+	var className = attrs.className == null && attrs.class !== undefined ? attrs.className || attrs.class : attrs.class || attrs.className
```

Comment:
Note: this is actually equivalent to `className = attrs.class || attrs.className` if you split it into an equivalent `if`/`else` and play with it a little (negating conditions and inverting branches).

Reply:
I can adjust if the basis of this PR fit specs.